### PR TITLE
UI & Langs Rework

### PR DESCRIPTION
**Pull request by skmedix**

### CODE REVIEW

File: assets/launcher/lang/sklmessages_en_US.properties [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
 sidebar.browse=Open Profile Directory
 sidebar.delete=Delete Profile
-sidebar.profiles=Installed Profiles
+sidebar.profilelist=Profiles
 sidebar.loading=Loading Profiles...
 sidebar.settings=Launcher Settings
 sidebar.player=Player
 sidebar.guest=Welcome, guest! Please log in.
 sidebar.versions=Loading versions...
+sidebar.version.latest=Latest Release
```
Comment (Greg-21) [inline assets/launcher/lang/sklmessages_en_US.properties]: Didn't you mean "Latest Version" here, like in `editor.latest`?

**Comment by Greg-21** [comment]
If you know how to resolve these conflicts, please do it. I tried to do it myself, but after that the problem became more complicated (I think), so I had to perform a force push to bring this branch back.

And I hope you will squash all commits into one and then merge it without the default description which points all commit messages. Let this chaos in commits become forgotten :D

**Comment by skmedix** [comment]
No worries. There is still much to do on this PR

File: assets/launcher/lang/sklmessages_en_US.properties (hunk 1 above)
Comment (Greg-21) [inline assets/launcher/lang/sklmessages_en_US.properties]: Didn't you mean "Latest Version" here, like in `editor.latest`?

Also, while you are working on an overhauled UI, it sounds like a good time to do something with string `settings.lang.help` and `settings.lang.link`. Maybe the idea from [#71](https://github.com/skmedix/SKlauncher/pull/71#discussion_r205518956)?

Comment (skmedix) [reply assets/launcher/lang/sklmessages_en_US.properties]: Yes, it's same as `editor.latest`. But it's a different category so, I has to add it again.

Also thanks for reminding me #71

**Comment by skmedix** [comment]
@Greg-21 Could you help me with updating the rest of langs? Only one last commit is on the way, so new en_US is gonna be ready by the end of this week

**Comment by skmedix** [comment]
**Note:** Before merging we need to check:
#108 
https://github.com/roman-xaiko/SKlauncher/commit/88b778765c07b678e2e5d3e3c98fbac45ab62eab (Should be uk_UA instead of ua_UA)
https://github.com/VoRTeXTwEnTyFoUr/SKlauncher/commit/9ce03beb0e1770d1cf55426ac8e72862cb4aa25a (But it does some weird things like `game.status.play.offline` key, it's definitely not gonna fit the button, so it also needs some work)

**Comment by Greg-21** [comment]
@skmedix I can help you port all the existing lang files (other than U.S. English, of course) into the new schema once the U.S. English lang file is in the final form.

Also, what do you think about merging the Ukrainian and Romanian lang files into branch `master` "as they are" and then, once the U.S. English lang file is already in that branch, porting these two into the new schema?

**Comment by skmedix** [comment]
Good idea with uk_UA & ro_RO, we should do that

**Comment by skmedix** [comment]
I ran out of time, I'll try to finish that translation today

**Comment by skmedix** [comment]
**TODO:**
- [ ] sklmessages_cs_CZ.properties
- [ ] sklmessages_en_US.properties
- [ ] sklmessages_eo.properties
- [ ] sklmessages_es_ES.properties
- [ ] sklmessages_fr_FR.properties
- [ ] sklmessages_hu_HU.properties
- [ ] sklmessages_nb_NO.properties
- [ ] sklmessages_pl_PL.properties
- [ ] sklmessages_pt_BR.properties
- [ ] sklmessages_ro_RO.properties
- [ ] sklmessages_tr_TR.properties
- [ ] sklmessages_uk_UA.properties
- [ ] sklmessages_vi_VN.properties
- [ ] sklmessages_zh_CN.properties

**Comment by skmedix** [comment]
@NCPlyn @Waterrail @NightFurySL2001 @Gbr22 
Review your langs, pls

**Comment by NightFurySL2001** [comment]
Sorry but I would require help on zh-CN translations: I need a more precise differentiation for the terms in the launcher according to different situations. Mainly, I would need to seperate between the terms: player, user, profile. Any help would be appreciated. [problem mainly due to change in branch https://github.com/skmedix/SKlauncher/pull/104/commits/7d6eb6b517fd9211e664f30827a09b84a705491d and mixed terms]

**Comment by skmedix** [comment]
User / Player = Account
Profile = Set of settings

**Comment by skmedix** [comment]
@Greg-21 you're on fire 🔥

**Comment by NightFurySL2001** [comment]
Sorry, problem with the merged version at pull/104/commits/ee5050191349d217dc45909fff49f617e2988f30. There are many strings that are cherry-picked wrongly and further check on zh-CN translation is required. Some of the changes didn't even occur in the original submitted version. Tq.

**Comment by skmedix** [comment]
LGTM. @Greg-21  if you think it's ready, feel free to merge

**Comment by Greg-21** [comment]
I think we could wait a while for the Chinese translation. Next we should squash all commits into one and then merge.

**Comment by Greg-21** [comment]
I think we should also wait for #115 and further actions that depend on it.

**Comment by skmedix** [comment]
Note: I want to drop new update in next week, so it needs to be merged before Monday

**Comment by Greg-21** [comment]
@Gbr22 @Waterrail @TrueEL @NightFurySL2001 
If you find some time, could you translate these two new strings before Monday?

**Comment by Greg-21** [comment]
@Gbr22 @Waterrail @TrueEL @NightFurySL2001 
If you find some time, could you translate these two new strings before Monday?

**Comment by Greg-21** [comment]
I would also consider updating the Codeowners file. Since this file works as expected only for users with write permissions for the SKlauncher repository, we can consider removing from the list that people, who haven't been active here (or in the Discord server) for a very long time and adding those, who are active (like e.g. @NightFurySL2001), and then commenting out the strings with people you still don't trust or just don't want to send them invitation to collaboration.

**Comment by Gbr22** [comment]
I can translate the new strings later today

**Comment by LorenzoMontanoPecoraro** [comment]
I'll traslate the new lines  later today

**Comment by NightFurySL2001** [comment]
Lines are translated.

**Comment by Greg-21** [comment]
What do you think about squashing all commits into one and then merging it with a commit message that summarizes our work made here? I propose something like this (but if you have a better idea for this message, let me know :) ):
```
Huge launcher language files rewrite

As a result of UI rework.

This commit also contains work of the following people:
* Gbr22 (Hungarian translation)
* Greg-21 (lots of small improvements)
* LorenzoMontanoPecoraro (Italian translation)
* NightFurySL2001 (Chinese translation)
* TrueEL (Vietnamese translation)
* Waterrail (Polish translation)
```
The first line above doesn't exceed 50 characters, as GitHub suggest, and should go into commit title. Starting from 3rd line and all way to the bottom should go into the commit extended description.

@Gbr22 @LorenzoMontanoPecoraro Since you name yourself in line `about.translation.authors` in additional way, I didn't know which name should I include above, in the example commit message, so I used the GitHub's ones.

**Comment by Greg-21** [comment]
What do you think about squashing all commits into one and then merging it with a commit message that summarizes our work made here? I propose something like this (but if you have a better idea for this message, let me know :) ):
```
Huge launcher language files rewrite

As a result of UI rework.

This commit also contains work of the following people:
* Gbr22 (Hungarian translation)
* Greg-21 (lots of small improvements)
* LorenzoMontanoPecoraro (Italian translation)
* NightFurySL2001 (Chinese translation)
* TrueEL (Vietnamese translation)
* Waterrail (Polish translation)
```
The first line above doesn't exceed 50 characters, as GitHub suggests, and should go into commit title. Starting from 3rd line and all way to the bottom should go into the commit extended description.

@Gbr22 @LorenzoMontanoPecoraro Since you name yourself in line `about.translation.authors` in additional way, I didn't know which name should I include above, in the example commit message, so I used the GitHub's ones.

**Comment by Greg-21** [comment]
What do you think about squashing all commits into one and then merging it with a commit message that summarizes our work made here? I propose something like this (but if you have a better idea for this message, let me know :) ):
```
Huge launcher language files rewrite

As a result of UI rework.

This commit also contains work of the following people:
* Gbr22 (Hungarian translation)
* Greg-21 (lots of small improvements)
* LorenzoMontanoPecoraro (Italian translation)
* NightFurySL2001 (Chinese translation)
* TrueEL (Vietnamese translation)
* Waterrail (Polish translation)
```
The first line above doesn't exceed 50 characters, as GitHub suggests, and should go into commit title. Starting from 3rd line and all way to the bottom should go into the commit extended description.

@Gbr22 @LorenzoMontanoPecoraro Since you name yourself in line `about.translation.authors` in additional way, I didn't know which names should I include above, in the example commit message, so I used the GitHub's ones.

**Comment by Greg-21** [comment]
@Waterrail If you have time, you can still update Polish translation to include these two new strings :)

**Comment by Waterrail** [comment]
> @Waterrail If you have time, you can still update Polish translation to include these two new strings :)

Oh yeah right, I'm on it

**Comment by Waterrail** [comment]
Done.

**Pull request merged**